docs/en/getting-started.md

```diff
@@ -1 +1,154 @@
 # Getting Started
+
+Getting up and running with the official `vue-test-utils` package is straight forward.
+Use either `npm` or `yarn` to install the package:
+
+```shell
+# npm
+$ npm install --save vue-test-utils
+
+# yarn
+$ yarn add vue-test-utils
+```
+
+Let's test a simple Vue component like a counter to get a feeling for how to use these utils.
+
+```js
+// counter.js
+
+export default {
+  template: `
+    <div>
+      {{ count }}
+      <button @click="increment">Increment</button>
+    </div>
+  `,
+
+  data () {
+    return {
+      count: 0
+    }
+  },
+
+  methods: {
+    increment () {
+      this.count++
+    }
+  }
+}
+
+```
+
+This component is very basic in it's capabilities and scope,
+but never the less it allows for things we could test here. The general concept of interacting with a Vue component through the `vue-test-utils` is using a wrapper. This wrapper comes with a lot of useful helpers under its belt that should make writing even complex specs a breeze. But let's start simple:
+
```

Comment:
There should probably be a section on setting up with JSDOM/ running the tests in a browser. As they won't work if not in a browser